critiquebrainz/frontend/templates/macros.html

```diff
@@ -121,3 +121,8 @@
       {{ spotify_player(spotify_release_group_id, spotify_mappings, spotify_show_add_message) }}
   {% endif %}
 {% endmacro %}
+
+{% macro show_avg_rating(rating, count) %}
+  <i class="glyphicon glyphicon-star"></i>
+  <span style="font-size: 16px;">{{ rating }}</span><em class="text-muted">/5 {{ _('based on') }} {{ count }} {{ _('ratings') }}</em>
```

Comment:
`{{ _('based on') }} {{ count }} {{ _('ratings') }}` should be combined into one localisable string. It can be a bad idea that the order of words would be the same in all languages.

```python
_('based on %(number)s ratings', number=count)
```

Reply:
Combined into one.